lib/CL/clEnqueueNDRangeKernel.c

```diff
@@ -546,13 +546,13 @@ if (local_##c1 > 1 && local_##c1 <= local_##c2 && local_##c1 <= local_##c3 && \
   command_node->type = CL_COMMAND_NDRANGE_KERNEL;
   command_node->command.run.kernel = kernel;
   command_node->command.run.pc = pc;
-  command_node->command.run.local_x = local_x;
-  command_node->command.run.local_y = local_y;
-  command_node->command.run.local_z = local_z;
+  command_node->command.run.pc.local_size[0] = local_x;
+  command_node->command.run.pc.local_size[1] = local_y;
+  command_node->command.run.pc.local_size[2] = local_z;
```

Comment:
Just minor nitpick: it'd be slightly nicer to use "pc.local_size[0] = local_x;" etc on line 544, since line 548 copies the entire struct into command node.

Reply:
I did the other way around to avoid the useless pc copy.

Reply:
OK